exercises/evaluation/ex_rnw/ex_of_uf.Rnw

```diff
@@ -0,0 +1,13 @@
+\begin{enumerate}[a)]
+  \item For the following situations, state whether you would rather expect 
+  under- or overfitting behavior.
+  \begin{enumerate}[i)]
+    \item Large training set \& linear regression learner
+    \item Noisy training data \& high-degree polynomial regression learner
+    \item Few observations of many features \& linear regression learner
+    \item Data with class overlap \& LDA learner
```

Comment:
QDA würde aber ja genauso wenig perfekt sein, oder? Außerdem hat man ja egtl. immer class overlap in der Realität?